### DISCUSSION TITLE
这种柱形图是如何实现的呢？

### DISCUSSION THREAD

**Discussion by zldyqhz** [XCharts-Team/XCharts#266] [Q&A]
每一个x轴上三个数据，而且在同一个轴上
![1f9c72d55198d3d58a7f12fbd5f5b26](https://github.com/XCharts-Team/XCharts/assets/89789797/054e032b-c7a4-4618-8bfb-93f1222baa62)

**Comment by monitor1394** [comment]
设置serie的stack为同样的值会堆叠在一个柱子上。
